packages/core/core/src/Parcel.js

```diff
@@ -319,7 +319,7 @@ export default class Parcel {
         ),
         buildTime: Date.now() - startTime,
         requestBundle: async bundle => {
-          let bundleNode = bundleGraph._graph.getNode(bundle.id);
+          let bundleNode = bundleGraph._graph.getNodeByContentKey(bundle.id);
```

Comment:
Seeing all of these changes makes me wonder if `getNode` could rather be `getNodeByContentKey` instead and then there's `getNodeByNodeId`.